__tests__/app.test.js

```diff
@@ -66,6 +67,32 @@ describe('/api/articles/:article_id', () => {
     })
 });
 
+describe('/api/articles', () => {
+    test('GET 200: Returns an array of articles with a comment count column added', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+            const { articles } = response.body
+            expect(articles).toHaveLength(13)
+            expect(articles).toBeSorted({ key: 'created_at', descending: true})
+            articles.forEach((article) => {
+                expect(article).toMatchObject({
+                    article_id: expect.any(Number),
+                    title: expect.any(String),
+                    topic: expect.any(String),
+                    author: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    comment_count: expect.any(String)
```

Comment:
I would recommend having this come back as a number. As its for a count one would reasonably expect a number data type. You may have to update your psql query to do this (not by much)